tests/system/talloc3.nim

```diff
@@ -0,0 +1,74 @@
+discard """
+  targets: "c"
+"""
```

Comment:
```suggestion
  description: '''
    Using the higher-level allocation procedures with types that have a size not
    known at the NimSkull side must work
  '''
"""
```
In addition, please rename the test to something more descriptive (maybe `talloc_with_unknown_size.nim`).